src/external/lib/view-engine/filters/index.js

```diff
@@ -2,7 +2,9 @@ module.exports = {
   ...require('../../../../shared/view/nunjucks/filters/abstraction-period'),
```

Comment:
We are using a module `app-module-path` which is supposed to resolve the `/src` directory.

So you may find you can rewrite these as e.g.
`require('shared/view/nunjucks/filters/abstraction-period')`